openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -546,6 +546,16 @@ def fill_statement_lines_with_no_move(env):
         st_line.move_id.with_context(skip_account_move_synchronization=True).write(
             to_write
         )
+    if st_lines:
+        openupgrade.logged_query(
+            env.cr,
+            """
+            UPDATE account_move am
+            SET partner_bank_id = COALESCE(absl.bank_account_id, am.partner_bank_id)
+            FROM account_bank_statement_line absl
+            WHERE am.statement_line_id IN %s""",
```

Comment:
Why not include the value in https://github.com/OCA/OpenUpgrade/pull/3150/files#diff-c7c0a45514483f4ef6990fbaa0404370e1205cb5be272c0e87d4319199526d17R515 ?

Reply:
because absl.bank_account_id doesn't exist in the environment. it is a deprecated field.

Reply:
Then write just the needed ones:

```suggestion
            UPDATE account_move am
            SET partner_bank_id = absl.bank_account_id
            FROM account_bank_statement_line absl
            WHERE am.statement_line_id IN %s AND absl.bank_account_id IS NOT NULL AND am.partner_bank_id IS NULL""",
```